flake.nix

```diff
@@ -259,8 +243,6 @@
               config.packages.stub
               config.packages.tool
             ];
-
-            TEST_SYSTEMD = pkgs.systemd;
```

Comment:
We should keep this around, otherwise you can't run the lzbt tests from the shell anymore.

Reply:
Good point. `cargo test` in lzbt passes now and I've also left openssl and sbctl.